.github/workflows/devbuild.yml

```diff
@@ -422,17 +448,17 @@ jobs:
         run: |
           echo "github.event_name: ${{ github.event_name }}"
 
-      - name: install qt
+      - name: Install Qt
```

Comment:
It's not right to change from lower case to capitalization.  Look around, all names are using lower cases.

Please fix the style.